pyamg/aggregation/aggregation.py

```diff
@@ -406,9 +413,10 @@ def unpack_arg(v):
         raise ValueError('Unrecognized symmetry.')
 
     if keep:
```

Comment:
Storing Cnodes when keep=True makes sense to me.  I actually just added this to a fork that I'm fiddling around with.